pkg/minikube/cluster/cluster.go

```diff
@@ -225,15 +227,32 @@ func SetupCerts(d drivers.Driver) error {
 }
 
 func createHost(api libmachine.API, config MachineConfig) (*host.Host, error) {
-	driver := virtualbox.NewDriver(constants.MachineName, constants.Minipath)
-	driver.Boot2DockerURL = config.MinikubeISO
+	var driver drivers.Driver
+
+	switch config.VMDriver {
+	case "virtualbox":
+		d := virtualbox.NewDriver(constants.MachineName, constants.Minipath)
+		d.Boot2DockerURL = config.MinikubeISO
+		driver = d
+	case "vmwarefusion":
+		d := vmwarefusion.NewDriver(constants.MachineName, constants.Minipath).(*vmwarefusion.Driver)
```

Comment:
Looks like the build is still failing:  `pkg/minikube/cluster/cluster.go:274: undefined: vmwarefusion.Driver`

I think this should be `.(*drivers.Driver)`